### PR TITLE
[Spritelab] Use spaces in aliases

### PR DESCRIPTION
**Pull request by ajpal**
Update the aliases to use spaces, not underscores. 
Also includes a script to update the metadata file for each animation. I have not changed the files on S3 yet, I will do that at the same time as merging this PR. I have run the script, though, and spot-checked that it is working correctly. 

This PR is just for Spritelab, but we will want the same change for Gamelab too. I'm going to do that separately, just to reduce the diff size.

## Links

- [spec]()
- [jira](https://codedotorg.atlassian.net/browse/STAR-1133)

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/p5lab/spritelab/spriteCostumeLibrary.json [hunk 1]
```diff
@@ -7858,25 +7858,25 @@
...
+    "orange monster": ["category_characters/orange_monster"],
+    "ore coal": ["category_obstacles/ore_coal"],
+    "ore diamond": ["category_obstacles/ore_diamond"],
+    "ore emerald": ["category_obstacles/ore_emerald"],
+    "ore gold": ["category_obstacles/ore_gold"],
+    "ore iron": ["category_obstacles/ore_iron"],
+    "ore ironalt": ["category_obstacles/ore_ironAlt"],
+    "ore ruby": ["category_obstacles/ore_ruby"],
+    "ore silver": ["category_obstacles/ore_silver"],
     "p5.play": ["category_characters/fly_bot"],
```
Comment (jmkulwik) [inline apps/src/p5lab/spritelab/spriteCostumeLibrary.json]: NIT: Should it be `p5 play` or `p5.play`?

File: tools/scripts/updateAnimationAliases.rb [hunk 2]
```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env ruby
+require 'json'
+require 'optparse'
+require_relative '../../lib/cdo/cdo_cli'
+include CdoCli
+
+#
+# This script is for updating the aliases in the metadata for each animation
+# using spriteCostumeLibrary as the source of truth.
+# Usage:
```
Comment (jmkulwik) [inline tools/scripts/updateAnimationAliases.rb:10]: THANK YOU For listing this out!!! :)

File: tools/scripts/updateAnimationAliases.rb [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+GAMELAB_ANIMATION_LIBRARY_PATH = './cdo-animation-library/gamelab/'.freeze
+
+class AliasUpdater
+  def initialize(options)
+    @options = options
+  end
+
+  def get_update_aliases_for_animation(all_aliases, animation_name)
+    aliases = []
+    all_aliases.each do |aliaz, animations|
```
Comment (jmkulwik) [inline tools/scripts/updateAnimationAliases.rb:32]: NIT: Should `aliaz` be `alias`?

File: tools/scripts/updateAnimationAliases.rb [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+      animation_name = animation_path_regex.match(animation_path)[1]
+      new_aliases = get_update_aliases_for_animation(all_aliases, animation_name)
+      write_updated_metadata(animation_name, new_aliases)
+    end
+  end
+end
+
+options = {}
+cli_parser = OptionParser.new do |opts|
+  opts.banner = "Usage: ./rebuildAnimationLibraryManifest.rb [options]"
```
Comment (jmkulwik) [inline tools/scripts/updateAnimationAliases.rb]: Still accurate?

File: tools/scripts/updateAnimationAliases.rb (hunk 3 above)
Comment (ajpal) [reply tools/scripts/updateAnimationAliases.rb:32]: `alias` is a keyword in Ruby so I copied this workaround from rebuildAnimationLibraryManifest.rb

File: tools/scripts/updateAnimationAliases.rb (hunk 4 above)
Comment (ajpal) [reply tools/scripts/updateAnimationAliases.rb]: fixed

File: tools/scripts/updateAnimationAliases.rb [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+end
+
+options = {}
+cli_parser = OptionParser.new do |opts|
+  opts.banner = "Usage: ./updateAnimationAliases.rb [options]"
+  opts.separator ""
+  opts.separator "Options:"
+
+  opts.on('-s', '--spritelab', 'Update aliases for Sprite Lab (default is Game Lab)') do
+    options[:spritelab] = true
```
Comment (jmkulwik) [inline tools/scripts/updateAnimationAliases.rb:71]: Would you mind listing this option in the "usage" comments at the top of the file too? I likely wouldn't have read this far if I was just running this to update the aliases.

**Review by jmkulwik — APPROVED** [review]
LGTM!

**Pull request merged**